quill-cassandra-macros/src/main/scala/io/getquill/context/cassandra/encoding/CollectionsEncodingMacro.scala

```diff
@@ -0,0 +1,39 @@
+package io.getquill.context.cassandra.encoding
+
+import io.getquill.util.Messages._
+
+import scala.reflect.macros.blackbox.{ Context => MacroContext }
+
+trait CollectionsEncodingMacro {
```

Comment:
Could you share some context on why you need these macros for cassandra? It's not clear to me